### PR TITLE
add Nida8930 to team.md

### PR DESCRIPTION
**Pull request by Nida8930**
N/A

### Description
I recently completed the first steps and added my user name to the team list

### Raw.Githack preview link
https://raw.githack.com/Nida8930/Nida8930.github.io/branch-internlist/#!pages/vi/team.md

### CODE REVIEW

**Review by pattanawadee88 — APPROVED** [review]

**Review by dogi — CHANGES_REQUESTED** [review]
on hold until interview

**Comment by dogi** [comment]
by the way @Nida8930 is "nedavis2" also you?

**Comment by Nida8930** [comment]
@dogi Yes it is, I'm not sure how it came up in here, its an old account not associated with my current school.

File: pages/vi/team.md [hunk 1]
```diff
@@ -19,5 +19,6 @@
 |[pattanawadee88](profiles/pattanawadee88.md)|2019-09-20|
 |[sriharivishnu](profiles/sriharivishnu.md)|2019-10-22|
 |[kjong](profiles/kjong.md)|2019-10-28|
+|[Nida8930](profiles/Nida8930.md)|2019-11-12|
```
Comment (dogi) [inline pages/vi/team.md:22]: @Nida8930 congrats

**Review by dogi — APPROVED** [review]

**Pull request merged**